pages/dkp/kommander/2.0/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: layout.pug
+beta: true
+navigationTitle: Kommander 2.0
+title: Kommander 2.0
+version: 2.0
+featureMaturity:
+enterprise: false
+category: K-Sphere
+menuWeight: 0
+---
+
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federation and cost management, across multiple clusters, whether they use a new Konvoy cluster or existing 3rd party/DIY distribution.
```

Comment:
@cmastr @dkoshkin we now can add this to the frontmatter and merge all the new things to make this PR disappear:

```
subtree:
  draft: true
```

any page with `draft: true` won't be rendered. every property within `subtree` will be inherited by the file itself and all children.